core_backend/tests/api/test_manage_content.py

```diff
@@ -303,6 +303,7 @@ def _search_qdrant_collection_by_id(self, content_id: uuid.UUID) -> List[Record]
         [
             ("title1", "content text 1", {}),
             ("title 1", "content text 2", {"meta_key": "meta_value"}),
+            ("title 1", "content text 2", {"meta_key": "meta_value"}),
         ],
```

Comment:
What's the purpose of this repeat?

Reply:
It's a mistake :'( Thank you for catching!